lib/hope-tab.js

```diff
@@ -115,11 +167,18 @@ window.HopeTab = fxosComponent.register('hope-tab', {
   activate() {
     this.setAttr('active', '');
     this.els.iframe.setVisible(true);
+    clearTimeout(this.showBarTimeout);
+    this.els.inner.style.overflowY = 'scroll';
```

Comment:
I just noticed that this add a noticeable delay when open in scrolling to the current tab. As the `.mutation()` task is blocked until scrolling has finished. Previously the overlay would be faded out  while scrolling was still in progress.